addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -173,9 +178,9 @@
                                     'inline-field': 'body_html'
                                 }"/>
                             </page>
-                            <page string="Options">
+                            <page string="Settings">
```

Comment:
Can we set a `name` on this element while we're at it? There is already a diff anyways and it'll allow easier xpaths.